volumes/miovision/api/readme.md

```diff
@@ -137,6 +141,18 @@ In command prompt, navigate to the folder where the python file is [located](../
 
 The `--pull` and `--agg` commands allow us to run data pulling and aggregation together or independently, which is useful for when we want to check out the data before doing any processing. For example, when we are [finding valid intersection movements for new intersections](../update_intersections/readme.md#update-miovision_apiintersection_movements).  
 
+### Alerts
+
+Although it it typically run daily through the Airflow DAG [miovision_pull](../../../dags/miovision_pull.py) `pull_alerts` task, you can also pull from the Alerts API using the command line. This is  helpful for backfilling multiple dates at once. An example command is: 
+`python3 pull_alert.py run-alerts-api-cli --start_date=2024-06-01 --end_date=2024-07-01`
```

Comment:
because the connection grabbing is in the pull_alert.py instead of the dag, we should mention this command can only be run in the airflow venv.

Reply:
there is an extra space between `this is` and `helpful`